website/static/data/showcases.json

```diff
@@ -43,7 +43,7 @@
   },
   {
     "caption": "嘀嗒出行",
-    "image": "showcase/didachuxing.jpg",
+    "image": "showcase/didachuxing.webp",
```

Comment:
Hi, you could visit https://caniuse.com/?search=webp to check if a property or feature is supported in different browsers.

![image](https://user-images.githubusercontent.com/2106987/158945642-95cabb4e-2ec7-43a3-8e82-29448208f6e0.png)


For webp, it seems that Safari doesn't support it well?

![image](https://user-images.githubusercontent.com/2106987/158945562-df409750-b19b-4974-ad79-154c70600b97.png)

Reply:
@juzhiyuan 
what about IE?
![image](https://user-images.githubusercontent.com/99874763/158946394-c52229af-50c0-4ee7-a3dc-fccf62a643c8.png)

Reply:
Microsoft drops IE, and I support not considering it.

Reply:
Truth be told, I prefer to use svg vector graphics over webp, if icons available.

Reply:
> Truth be told, I prefer to use SVG vector graphics over webp, if icons available.

I'm sorry about that.I have tried my best to find these icons of SVG, but it's too hard find them.And some of their website ICONS are not particularly clear.

Reply:
Yes, so I modified it and added the sentence `Hopefully we can make it happen in the future.`

Reply:
> Yes, so I modified it and added the sentence `Hopefully we can make it happen in the future.`

Yes, I hope so! If I find them， I will pull the request immediately.😄